### PR TITLE
parboiled-java with fix for JDK16+ added, min jdk changed to 11

### PR DESCRIPTION
**Pull request by bgalek**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -18,13 +18,14 @@ java {
...
     toolchain {
-        languageVersion.set(JavaLanguageVersion.of(8))
+        languageVersion.set(JavaLanguageVersion.of(11))
     }
 }
 
 dependencies {
     implementation("org.apache.commons:commons-lang3:3.12.0")
     implementation("org.parboiled:parboiled-java:1.3.1")
+    implementation(files("/Users/bartosz.galek/Work/opel/lib/parboiled-java-1.4.0.jar"))
```
Comment (tfij) [inline build.gradle.kts:28]: seriously? :)

File: src/main/java/pl/allegro/tech/opel/OpelParser.java [hunk 2]
```diff
@@ -5,10 +5,11 @@
 import org.parboiled.annotations.BuildParseTree;
 import org.parboiled.annotations.SuppressSubnodes;
 
+import java.lang.invoke.MethodHandles;
 import java.math.BigDecimal;
 
 @BuildParseTree
-class OpelParser extends BaseParser<OpelNode> {
+public class OpelParser extends BaseParser<OpelNode> {
```
Comment (tfij) [inline src/main/java/pl/allegro/tech/opel/OpelParser.java:12]: why it has to be public?

**Review by tfij — CHANGES_REQUESTED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (bgalek) [reply build.gradle.kts:28]: accident :D

File: src/main/java/pl/allegro/tech/opel/OpelParser.java (hunk 2 above)
Comment (bgalek) [reply src/main/java/pl/allegro/tech/opel/OpelParser.java:12]: this is a long story.... :P
jdk16 changed warning of illegal module access reflection to runtime error.
Parboiled is based on this behaviour :( I've made this branch to test my fix. Fortunately parboiled author came along and he is willing to merge my fix ;) package classes cannot be accessed in the way parboild works (org.parboiled cannot access pl.allegro.tech.opel class)

Comment (tfij) [reply src/main/java/pl/allegro/tech/opel/OpelParser.java:12]: wow!

**Review by tfij — APPROVED** [review]

**Pull request merged**